### DISCUSSION TITLE
[Office Hours] Getting a bunch of build errors + questions regarding Spezi Questionnaire

### DISCUSSION THREAD

**Discussion by kc224** [CS342/.github#69] [Office Hours]
### Related Discussion

N/A

### Reproduction

I am trying to build my application to test Firebase user authentication (which was working previously) but have started to get many build errors all of a sudden.

The first error is "Command SwiftCompile failed with a nonzero exit code" and the others are all "error opening file" messages.

<img width="280" alt="Screenshot 2024-02-21 at 1 06 59 AM" src="https://github.com/CS342/.github/assets/63617639/e82ba992-74fa-4fba-a83c-d87ec9350f71">

### Additional context

I've tried cleaning the build folder, along with restarting XCode and my computer. 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by kc224** [comment]
Fixed it!

**Reply by PSchmiedmayer** [reply]
Great to hear @kc224; can you share how you fixed your challenge?
